api/leaderboard.go

```diff
@@ -597,18 +583,12 @@ func (app *App) GetTopMembers(ctx context.Context, req *api.GetTopMembersRequest
 	if tenantID != nil {
 		result, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```

Comment:
Instead of having line 591, what about this?

```suggestion
		members, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```